clients/fides-js/src/lib/cookie.ts

```diff
@@ -131,7 +145,13 @@ export const getFidesConsentCookie = (): FidesCookie | undefined => {
  */
 export const getOrMakeFidesCookie = (
   defaults?: NoticeConsent,
-  fidesClearCookie: boolean = false,
+  {
+    fidesClearCookie = false,
+    fidesCookieSuffix,
+  }: {
+    fidesClearCookie?: boolean;
+    fidesCookieSuffix?: FidesInitOptions["fidesCookieSuffix"];
+  } = {},
```

Comment:
I would suggest combining Partial and Pick to be consistent with what's being passed in.
```suggestion
  }: Partial<
    Pick<FidesInitOptions, "fidesClearCookie" | "fidesCookieSuffix">
  > = {},
```